### PR TITLE
boards: remove support for wsn430-1_3b and wsn430-1_4

### PR DESCRIPTION
**Pull request by aabadie**
### Contribution description

The WSN430 boards are no longer available on the iot-lab testbed.

Thus, this PR:
- removes the support for both boards
- removes the mapping of these boards from the testbed support integration in the build system (via IOTLAB_NODE)
- removes all occurrences of the boards in applications' Makefile and Makefile.ci.

Regarding the change in LOSTANDFOUND: for the commit hash to be correct, I think it should be provided in a separate PR and the commit hash to reference the merge commit ?

### Testing procedure

- no more reference to wsn430/WSN430 in the codebase
- Integration of other iotlab boards still works

### Issues/PRs references

None

### CODE REVIEW

**Comment by miri64** [comment]
Maybe also add a deprecation note to the respective boards doc.

**Comment by aabadie** [comment]
> Maybe also add a deprecation note to the respective boards doc.

This PR is not removing them, just the integration with the `IOTLAB_NODE` variable. I'm finalizing the other PR that is actually removing them completely.

**Comment by aabadie** [comment]
I finally all removal of the wsn430 boards in this PR and updated the title.

File: LOSTANDFOUND.md [hunk 1]
```diff
@@ -173,3 +173,14 @@ Reason for removal:
 - its only user within the RIOT code base `gnrc_ipv6_ext` was reworked in
   f671a87fe2c539c3aecd595ae03fa4f6f209d042 so it is not needed anymore.
 - the function was deprecated in f2760c033c5f332be076b25aa212aca4007c3d65
+
+### boards/wsn430-v1_3b, boards/wsn430-v1_4 [4511eebbf897e1d3ea04235c8385276c75f0c3b9]
```
Comment (miri64) [inline LOSTANDFOUND.md:177]: That commit hash does not seem to point anywhere. You also used multiple commits, so multiple commits should be here.

Comment (aabadie) [reply LOSTANDFOUND.md:177]: In fact, I would prefer to provide this in a follow-up and have only commit that points to the merge commits of this PR.

Comment (miri64) [reply LOSTANDFOUND.md:177]: :+1: then lets do it like that, but please provide the follow-up as a depending PR before merging.

Comment (aabadie) [reply LOSTANDFOUND.md]: Done: #14495

**Pull request merged**